aptly_api/parts/snapshots.py

```diff
@@ -49,7 +49,16 @@ def create_from_repo(self, reponame: str, snapshotname: str, description: Option
         if description is not None:
             body["Description"] = description
 
-        resp = self.do_post("api/repos/%s/snapshots" % quote(reponame), json=body)
+        resp = self.do_post("api/repos/%s/snapshots" %
+                            quote(reponame), json=body)
```

Comment:
These newlines don't really fit with the style across the other files... is this just your style or something like `black`?

Reply:
They come from my IDE (auto formatted). I'm using Atom, unfortunately with default line lengths etc..I can fix that if needed.